datafusion/functions-aggregate/src/approx_percentile_cont.rs

```diff
@@ -31,20 +31,18 @@ use arrow::{
 use arrow_schema::{Field, Schema};
 
 use datafusion_common::{
-    downcast_value, internal_err, not_impl_err, plan_err, DFSchema, DataFusionError,
-    ScalarValue,
+    downcast_value, internal_err, not_impl_err, plan_err, DataFusionError, ScalarValue,
```

Comment:
```suggestion
    downcast_value, internal_err, not_impl_err, plan_err, DataFusionError, Result, ScalarValue,
```

Reply:
Somes changes I prefer to leave it alone to make this PR smaller, for example like the `Result` in this file